lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -363,3 +374,45 @@ def _assert_retry_not_called(self, mock_retry):
         Verifies the task was not retried.
         """
         self.assertFalse(mock_retry.called)
+
+
+@ddt.ddt
+class ComputeGradesForCourseTest(HasCourseWithProblemsMixin, ModuleStoreTestCase):
+    """
+    Test compute_grades_for_course task.
+    """
+
+    ENABLED_SIGNALS = ['course_published', 'pre_publish']
+
+    def setUp(self):
+        super(ComputeGradesForCourseTest, self).setUp()
+        self.set_up_course()
+        self.users = [UserFactory.create() for _ in xrange(12)]
+        for user in self.users:
+            CourseEnrollment.enroll(user, self.course.id)
+
+
+    @ddt.data(*xrange(0, 12, 3))
+    def test_behavior(self, batch_size):
+        result = compute_grades_for_course.delay(
+            course_key=six.text_type(self.course.id),
+            batch_size=batch_size,
+            offset=4
+        )
+        self.assertEqual(
+            PersistentSubsectionGrade.objects.filter(course_id=self.course.id).count(),
+            min(batch_size, 8)  # No more than 8 due to offset
+        )
```

Comment:
Question for @edx/educator-neem devs: It looks like PersistentCourseGrade objects aren't getting created.  Is there still a toggle for that which needs to get turned on for the test?